### PR TITLE
New Game: Matching_Pair_Game

### PR DESCRIPTION
**Pull request by priyashuu**
## PR Description 📜

he Matching Pairs Game is a fun and engaging memory game developed using HTML, CSS, and JavaScript. The game challenges players to match pairs of cards with identical images within a limited time. It offers a simple yet addictive gameplay experience, enhanced with a sleek user interface .
Fixes # <4844>

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
Matching_Pair_Game
## workingVideo
https://github.com/user-attachments/assets/761e65a1-6ef8-49ec-ac12-574ea1e15555

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @priyashuu ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -857,9 +857,7 @@ This repository also provides one such platforms where contributers come over an
 | [UNO game with computer](https://github.com/kunjgit/GameZone/tree/main/Games/UNO_game_with_Computer) |
 |[Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/main/Games/Dice_rolling_simulator)|
 |[Space_Dominators](https://github.com/kunjgit/GameZone/tree/main/Games/Space_Dominators)|
-| [Simon_Says](https://github.com/kunjgit/GameZone/tree/main/Games/Simon_Says) |
```
Comment (ishita-43) [inline README.md:860]: these game entry are missing from your file simon says, idle  miner, five nights at freddys
make sure your file is in sync with latest changes

File: README.md [hunk 2]
```diff
@@ -857,9 +857,7 @@ This repository also provides one such platforms where contributers come over an
...
-|[Idle_miner](https://github.com/kunjgit/GameZone/tree/main/Games/Idle_miner)|
-|[Five_Nights_at_Freddys](https://github.com/kunjgit/GameZone/tree/main/Games/Five_Nights_at_Freddys)|
+|[Matching_Pair](https://github.com/priyashuu/GameZone/tree/main/Games/Matching_pair)
```
Comment (ishita-43) [inline README.md:860]: change the your id name with kunjgit to make changes visible on the site as other entries do

**Review by ishita-43 — CHANGES_REQUESTED** [review]
You need to check following changes in your PR

**Comment by ishita-43** [comment]
Resolve merge conflicts

**Comment by priyashuu** [comment]
okay Done ...

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by priyashuu** [comment]
Thank you ⭐🫡❤️

**Comment by kunjgit** [comment]
Thank you  @priyashuu , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**